apps/experiments/models.py

```diff
@@ -744,55 +744,81 @@ def format_tools(tools: set):
         return Version(
             instance=self,
             fields=[
-                VersionField(group_name="Group 1", name="description", raw_value=self.description),
-                VersionField(group_name="Group 1", name="prompt_text", raw_value=self.prompt_text),
-                VersionField(group_name="Group 1", name="llm_model", raw_value=self.llm),
-                VersionField(group_name="Group 1", name="llm_provider", raw_value=self.llm_provider),
+                VersionField(group_name="General", name="description", raw_value=self.description),
```

Comment:
This is what the user will see now:

![image](https://github.com/user-attachments/assets/cd509e9a-385e-4e18-9071-98ef4ddecb49)

![image](https://github.com/user-attachments/assets/0b2bd107-29f3-4a45-8417-ad50fd376a7d)